docs/billing.md

```diff
@@ -10,139 +10,171 @@ stack:         all
+ 13 14 15 16 17 18 19      -  -  -  $  $  $  $  <- the  16th is on a Thursday
+ 20 21 22 23 24 25 26      $  $  $  $  $  $  $
+ 27 28 29 30               $  $  $  $
+```
+
+The dollar-sign indicates the days you would be billed for if signe up on the 16th.
 
 
 ### Monthly invoices
 
-Within the first days of every new month you will get the invoice for your last month's consumption. After receiving the invoice, the payments are due and will automatically be done by us. Credit Cards usually will happen within the first days of the month, SEPA direct within the first two weeks.
+Within the first few days of every new month, you will get the invoice for the consumption of the last month. After receiving the invoice, the payments are due and are charged automatically. Credit Cards payments will usually happen within the first few days of the month. SEPA direct payment will be charged within the first two weeks.
 
 
-### The monthly billing cycle in detail
+#### The monthly billing cycle in detail
 
-At the end of the month, the fortrabbit billing bot will go through all the Billing Contacts and check for the currents month usage. Based on that data, the monthly invoices will be created and made accessible from the Dashboard. The invoice is dated on the last day of the month and the service period is that month. On the first of the (next) month, the fortrabbit mail bot will send an invoice notice by e-mail to all clients. On the 3rd of the month, the fortrabbit billing bot will attempt to charge for all invoice with payment method credit card. Around the 7th of the month the SEPA direct payments for the invoices, for clients with that payment method will be issued and a few days later executed.
+The fortrabbit billing bot calculates the due amount for each Billing Contact at the end of the month. Based on that data, a new invoice will be created. The invoices are accessible through our Dashboard. The invoice is dated to the last day of the month and the service period is that month.
 
-At the middle of the month, a human will look after all the bounced invoices. That human also will attempt to issue payment again and will also might write addition payment reminders.
+* On the first day of the following month, all clients are notified about this new invoice.
+* On the 3rd of the month, the fortrabbit billing bot will charge all invoices having credit card as the payment method.
+* Around the 7th of the month, all the SEPA direct payments invoices are issued and executed a few days later.
+
+In the middle of a month, a human will look over bounced invoices. At that time, due payments will be re-charged and additional payment reminders may be sent out.
 
 
 ### Costs monitoring
 
-You can check the current costs with your Billing Contact, see [here](/billing-contact#toc-costs-monitoring).
+You can check the current costs by looking at your *Billing Contact*, see link:/billing-contact#toc-costs-monitoring[here].
```

Comment:
`link://` is that valid Markdown?

Reply:
I did not invent link:// it's used other places in this document.

Reply:
My bad. It was actually PHP-Storm that messed me up